FABulous/fabric_definition/Tile.py

```diff
@@ -309,3 +310,115 @@ def globalConfigBits(self) -> int:
             ret += b.configBit
 
         return ret
+
+    def get_min_die_area(
```

Comment:
Can this be combined with the Supertitle get_min_die_area to avoid duplications?